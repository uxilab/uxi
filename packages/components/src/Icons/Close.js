import React from 'react';
import PropTypes from 'prop-types';
import SvgIcon from '../SvgIcon';
/* eslint-disable max-len */

const Close = props => (
  <SvgIcon {...props}>
    <svg
      focusable="false"
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 16 16"
      width="24px"
      height="24px"
    >
      <g id="Layer_2" data-name="Layer 2">
        <g id="Layer_1-2" data-name="Layer 1">
          <polygon points="16 1.3 14.7 0 8 6.7 1.3 0 0 1.3 6.7 8 0 14.7 1.3 16 8 9.3 14.7 16 16 14.7 9.3 8 16 1.3" />
        </g>
      </g>
    </svg>
  </SvgIcon>
);

Close.propTypes = {
  color: PropTypes.string,
};

Close.defaultProps = {
  color: '#6d6d71',
};

export default Close;
