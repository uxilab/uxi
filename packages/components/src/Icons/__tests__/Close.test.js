import React from 'react';
import Close from '../Close';

describe('Icon : <Close />', () => {
  it('match snapshot', () => {
    expect(shallow(<Close />)).toMatchSnapshot();
  });
});

