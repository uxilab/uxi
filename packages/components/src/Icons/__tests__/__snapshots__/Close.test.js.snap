// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Close /> match snapshot 1`] = `
<SvgIcon
  color="#6d6d71"
  hoverColor="#6d6d71"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 16 16"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <polygon
          points="16 1.3 14.7 0 8 6.7 1.3 0 0 1.3 6.7 8 0 14.7 1.3 16 8 9.3 14.7 16 16 14.7 9.3 8 16 1.3"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
